modules/ROOT/pages/Development/Satisfactory/Audio.adoc

```diff
@@ -7,14 +7,215 @@ This page is a work in progress.
+
+image:Satisfactory/Wwise/Wwise_Image9.png[Wwise_Image9, 600]
+
+Now, it's finally time to generate the Soundbank for Unreal Engine.
+
+In the Project Explorer, on the Soundbanks tab, right-click on the soundbank and click `Generate Soundbank(s) for all platforms`.
+
+image:Satisfactory/Wwise/Wwise_Image10.png[Wwise_Image10, 500]
+
+=== Creating AKEvents In Unreal Engine
+
+Open the Unreal Engine project settings and Navigate to the Wwise integration settings.
+
+Change the `Root Output Path` setting to `<WwiseProjectPath>/GeneratedSoundBanks`.
+
+Then Change the `Unreal Audio Routing` to `Both Wwise and Unreal audio`
+
+You may need to restart Unreal after these changes.
+
+Once your audio, event, and soundbanks are created, it's time to utilize them in the Unreal Engine.
+
+Create a new Audiokinetic Event by right-clicking on an empty area in the Content Browser.
+
+Go To `Audiokinetic -> Audiokinetic Event`
+
+image:Satisfactory/Wwise/Wwise_Image11.png[Wwise_Image11, 500]
+
+Name the new asset **Exactly** the same name as the event in Wwise.
+
+[NOTE]
+====
+If the asset name is exactly the same as the Wwise event, Unreal Engine will automatically populate the correct event information to the variables below.
+====
+
+Verify that the event has the correct information by opening the event in Unreal.
+
+* Verify that the `Wwise Short ID` matches the event ID in Wwise.
+* Verify the `Wwise Name` matches the event name.
+
+image:Satisfactory/Wwise/Wwise_Image12.png[Wwise_Image12, 500]
+
+If everything has worked correctly, you can now test the event by right-clicking the event in Unreal and clicking `Play Event`
+
+=== Playing AKEvents In Blueprints
+
+==== Fire & Forget Events
+
+If you would like to play a one-off AK event, such as a spark sound, you will need to use the following method in your blueprint.
+
+image:Satisfactory/Wwise/Wwise_Image13.png[Wwise_Image13, 100%]
+
+This will first create a new `AK Component` at the blueprint's root component and then play the AK Event specified.
+
+Once the AK event has ended, you must clean up the `AK Component` using the callback event.
+
+==== Looping Events
+
+If you would like to play a sound effect that might loop, for example, a humming/buzzing sound for a building.
+
+The best method is to add an `AK Component` to your blueprint and set the `AK Audio Event` variable on the component.
+
+You must ensure that you stop the event when the actor is destroyed.
+
+image:Satisfactory/Wwise/Wwise_Image14.png[Wwise_Image14, 100%]
+
+
+=== Setting Up Attenuation
+
+If you plan to use Wwise audio for buildings, you will need to set up attenuation. By default, the audio will play even if you are on the other side of the map.
+
+Wwise has a great tutorial on how to set this up here: https://youtu.be/lOP0seni1DY[Wwise & Unreal | Adding 3D Audio With Attenuation]
+
+=== Updating Wwise Values From Your Mod
+
+You can update audio values using RTPC (Game Parameters).
```

Comment:
what does RTPC stand for? it sounds like Game Parameters is part of the Unreal asset side of things

Reply:
Updated with better explaination on what RTPC means and clearer understanding of what it does.